frontend/src/app/dashboard/dashboard.component.html

```diff
@@ -0,0 +1,34 @@
+<div class="dashboard-container">
+  <p class="headline-large">Ziele und Resultate</p>
+
+  <div>
+    <mat-form-field appearance="fill" class="filter-dropdown">
+      <mat-label>Team</mat-label>
+      <mat-select
+        [formControl]="teams"
+        multiple
+        (selectionChange)="changeTeamFilter($event.value)"
+      >
+        <mat-option
+          class="team-dropdown"
+          *ngFor="let team of teamList | async"
+          [value]="team.name"
+          >{{ team.name }}</mat-option
+        >
+      </mat-select>
+    </mat-form-field>
+
+    <mat-form-field appearance="fill">
+      <mat-label>Zyklus</mat-label>
```

Comment:
Verwende statt Zyklus direkt Quarter. Im Label und auch in allen Variablen sowie Funktionen. Figma stimmt hier nicht. Das Domain-Modell ist korrekt.